spec/spec_helper.rb

```diff
@@ -64,7 +64,9 @@ def peach_with_index(*, &y)
           end
         end
 
-        # insert the cassette recording everything what happens outside the tests cases
+        # Insert cassette for before(:all) and after(:all) blocks.
+        # vcr_all_cassette specifies the cassette name. Unique name is needed
```

Comment:
Why is this neccessary? Is it not simpler to use a unique string in the "describe"?